CondFormats/EcalObjects/interface/EcalLiteDTUPedestals.h

```diff
@@ -0,0 +1,47 @@
+#ifndef EcalLiteDTUPedestals_h
+#define EcalLiteDTUPedestals_h
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include "CondFormats/EcalObjects/interface/EcalCondObjectContainer.h"
+#include "CondFormats/EcalObjects/interface/EcalConstants.h"
+
+//const int NGAINS = 2;
+
+//const int gainId1 = 1;
+//const int gainId10 = 0;
+
+class EcalLiteDTUPed {
```

Comment:
I would rather call the class like the file, I think it is more understandable (and similar to the Phase1 "EcalPedestals.h). It is just a bit tricky for this typedef we use (also "inspired" to the Phase1 code...):
`typedef EcalLiteDTUPedestalsMap EcalLiteDTUPedestals;`
My proposed solution is to remove this typedef and to use EcalLiteDTUPedestalsMap in the code, instead of EcalLiteDTUPedestals. It is probably even more straightforward

Reply:
Calling the class like the file is also fine with me if you can resolve the naming conflict with the typedef.